apps/remix-ide/src/app/components/plugin-manager-component.js

```diff
@@ -84,112 +22,90 @@ const profile = {
 class PluginManagerComponent extends ViewPlugin {
   constructor (appManager, engine) {
     super(profile)
-    this.event = new EventEmitter()
+    // this.event = new EventEmitter() // already exists in engine so not needed here
     this.appManager = appManager
+    this.engine = engine
+    this.pluginManagerSettings = new PluginManagerSettings()
+    this.htmlElement = document.createElement('div')
+    this.htmlElement.setAttribute('id', 'pluginManager')
     this.views = {
       root: null,
       items: {}
     }
-    this.localPlugin = new LocalPlugin()
     this.filter = ''
-    this.appManager.event.on('activate', () => { this.reRender() })
-    this.appManager.event.on('deactivate', () => { this.reRender() })
-    this.engine = engine
-    this.engine.event.on('onRegistration', () => { this.reRender() })
+    this.pluginNames = this.appManager.actives
+    this.activePlugins = []
+    this.inactivePlugins = []
+    this.activeProfiles = this.appManager.actives
+    this._paq = _paq
+    this.listenOnEvent()
   }
 
+  /**
+   * Checks and returns true or false if plugin name
+   * passed in exists in the actives string array in
+   * RemixAppManager
+   * @param {string} name name of Plugin
+   */
   isActive (name) {
-    return this.appManager.actives.includes(name)
+    this.appManager.actives.includes(name)
```

Comment:
I think `return` is needed

Reply:
Mistakenly removed the return. It is replaced.